.github/workflows/main.yml

```diff
@@ -20,72 +20,1307 @@ name: NetBeans
 on:
   push:
   pull_request:
+    # unlocked event is used as super secret restart button
+    types: [opened, synchronize, unlocked]
```

Comment:
"Lock Conversation" button is on the right panel on PRs. Unlock will start a fresh run without having to sync. I don't think we will need this very often, but its good to have that option.

We are using some fairly new github features, until recently you couldn't restart individual jobs at all. I am sure there will be better ways to implement this one day.